databricks/koalas/tests/test_dataframe.py

```diff
@@ -2263,13 +2263,23 @@ def _test_cumprod(self, pdf, kdf):
         self.assert_eq(pdf.cumprod().sum(), kdf.cumprod().sum(), almost=True)
 
     def test_cumprod(self):
-        pdf = pd.DataFrame(
-            [[2.0, 1.0], [5, None], [1.0, 1.0], [2.0, 4.0], [4.0, 9.0]],
-            columns=list("AB"),
-            index=np.random.rand(5),
-        )
-        kdf = ks.from_pandas(pdf)
-        self._test_cumprod(pdf, kdf)
+        if LooseVersion(pyspark.__version__) >= LooseVersion("2.4"):
+            pdf = pd.DataFrame(
+                [[2.0, 1.0, 1], [5, None, 2], [1.0, 1.0, 3], [2.0, 4.0, 4], [4.0, 9.0, 5]],
+                columns=list("ABC"),
+                index=np.random.rand(5),
+            )
+            kdf = ks.from_pandas(pdf)
+            self._test_cumprod(pdf, kdf)
+        else:
+            with self.sql_conf({"spark.sql.execution.arrow.enabled": False}):
```

Comment:
Why is this needed?

Reply:
Think it was my mistaken. Thanks for the point out